pkg/kubelet/kubelet.go

```diff
@@ -2313,6 +2376,99 @@ func (kl *Kubelet) HandlePodSyncs(pods []*v1.Pod) {
 	}
 }
 
+func (kl *Kubelet) canResizePod(pod *v1.Pod) (bool, *v1.Pod, v1.PodResizeStatus) {
+	var otherActivePods []*v1.Pod
+
+	node, err := kl.getNodeAnyWay()
+	if err != nil {
+		klog.ErrorS(err, "getNodeAnyway function failed")
+		return false, nil, ""
+	}
+	cpuAvailable := node.Status.Allocatable.Cpu().MilliValue()
+	memAvailable := node.Status.Allocatable.Memory().Value()
+	cpuRequests := resource.GetResourceRequest(pod, v1.ResourceCPU)
+	memRequests := resource.GetResourceRequest(pod, v1.ResourceMemory)
+	if cpuRequests > cpuAvailable || memRequests > memAvailable {
+		klog.V(2).InfoS("Resize is not feasible as request exceeds available node resources", "Pod", pod.Name)
```

Comment:
V=2 is much too high for these loglines, I would expect this to be V=3 at most. Same comment for other V=2 loglines below.

Reply:
Fixed in latest commit, please review.